src/main/java/org/opensearch/security/tools/democonfig/SecuritySettingsConfigurer.java

```diff
@@ -149,7 +149,7 @@ void updateAdminPassword() {
 
             // if ADMIN_PASSWORD is still an empty string, it implies no custom password was provided. We exit the setup.
             if (Strings.isNullOrEmpty(ADMIN_PASSWORD)) {
-                System.out.println("No custom admin password found. Please provide a password.");
+                System.out.println("No custom admin password found. Please provide a password via the environment variable OPENSEARCH_INITIAL_ADMIN_PASSWORD.");
```

Comment:
```suggestion
                System.out.println("No custom admin password found. Please provide a password by setting the environment variable OPENSEARCH_INITIAL_ADMIN_PASSWORD. Example: export OPENSEARCH_INITIAL_ADMIN_PASSWORD=<password>");
```

Reply:
I'm ok with suggesting something like this, but do we need to account for possibility of running on windows/different distributions?

Reply:
Good point. Leave it generic.